src/main/java/com/github/novelrt/fumocement/PointerOperations.java

```diff
@@ -0,0 +1,103 @@
+// Copyright © Matt Jones and Contributors. Licensed under the MIT License (MIT). See LICENCE.md in the repository root for more information.
+
+package com.github.novelrt.fumocement;
+
+/**
+ * Contains tools to manipulate pointers, mainly used for going through an array.
+ */
+public final class PointerOperations {
+  /**
+   * The size of the C {@code size_t} type, which is the value of {@code sizeof(size_t)}.
+   * <p>
+   * This value typically changes from 32-bit to 64-bit, and varies by the OS as well.
+   * It also represents the size of a pointer.
+   */
+  public static final int SIZE_T_SIZE;
+
+  static {
+    SIZE_T_SIZE = getNativeLongSize();
+  }
+
+  private PointerOperations() {
+  }
```

Comment:
Uhhhh?

Reply:
This is a java weirdness: there are no "static" classes like in C#, instead, you have to put a private constructor to prevent instantiations of that class.